packages/peregrine/lib/talons/AddressBookPage/addressBookPage.gql.js

```diff
@@ -22,14 +23,54 @@ export const GET_CUSTOMER_ADDRESSES = gql`
                 telephone
             }
         }
-
         countries {
             id
             full_name_locale
         }
     }
 `;
 
+/**
+ * We use the connection key directive here because Apollo will save
+ * this customer's PII in localStorage if not.
+ */
+export const ADD_NEW_CUSTOMER_ADDRESS = gql`
+    mutation AddNewCustomerAddressToAddressBook(
+        $address: CustomerAddressInput!
+    ) {
+        createCustomerAddress(input: $address)
+            @connection(key: "createCustomerAddress") {
+            id
+            city
+            country_code
+            firstname
+            lastname
+            postcode
+            region {
+                region
+                region_code
+                region_id
+            }
+            street
+            telephone
```

Comment:
We may be able to remove these fields from the query result since we do a `refetchQueries` to get the updated data.

Reply:
Removed from the `Add` mutation in https://github.com/magento/pwa-studio/pull/2879/commits/a10123de9d740b962336ba7496bce14cd78f6c93.